networkx/algorithms/centrality/group.py

```diff
@@ -49,27 +49,28 @@ def group_betweenness_centrality(G, C, normalized=True, weight=None):
       If None, all edge weights are considered equal.
       Otherwise holds the name of the edge attribute used as weight.
 
+    endpoints : bool, optional
+      If True include the endpoints in the shortest path counts.
+
     Raises
     ------
     NodeNotFound
        If node(s) in C are not present in G.
 
     Returns
     -------
-    betweenness : float
-       Group betweenness centrality of the group C.
+    betweenness : list of floats
```

Comment:
If there is only one value to return should we return a float (backward compatible) and for more than one we return a list of values? @rossbar, does that violate some principle?  What's the best way to transition to returning a collection of values instead of a single value?

Reply:
should I return the values in a dictionary in case of many groups, and as a float in case of single group in order to maintain backward compatible?

Reply:
I'm leery of using a dictionary because the keys could need to be a list or set of nodes. Can we return the Group Betweenness values in the same order as they appear in the original incoming list `C`?  Is that too confusing? 

Let's make it a single value if there is a single group and a list of values if `C` is a list of nodes... That is, create a flag variable in the `if thing in G:` clause to indicate scalar or list and then choose the output type based on that.

Reply:
I made the change. If C is a single group then return a single betweenness float value. If C is a list (even one containing a single group) the function returns a list of floats.